script/openqa-bootstrap-container

```diff
@@ -47,13 +54,23 @@ DeviceAllow=block-blkext rw
 WantedBy=machines.target
 EOF
 
-mkdir $CONTANER_PATH
-zypper -n --root $CONTANER_PATH addrepo $DEFAULT_REPO defaultrepo
-zypper -n --root $CONTANER_PATH --gpg-auto-import-keys refresh
-zypper -n --root $CONTANER_PATH install --no-recommends -ly $PKGS_TO_INSTALL
+if [ ! -d $CONTAINER_PATH ] ; then
+	mkdir -p $CONTAINER_PATH
+	zypper -n --root $CONTAINER_PATH addrepo $DEFAULT_REPO defaultrepo
+	zypper -n --root $CONTAINER_PATH --gpg-auto-import-keys refresh
+	# There are non-fatal errors when zyppering inside chroot, so ignoring
+	set +e
+	zypper -n --root $CONTAINER_PATH install --no-recommends -ly $PKGS_TO_INSTALL
```

Comment:
How about `zypper … ||:` to explicitly ignore the exit code instead of needing to tinker with the `+e` and `-e` above which would overwrite any external flag

Reply:
Good point, thank you.

Reply:
Or you could do `|| test $? == 123`

Reply:
I went for || test $? == 107 as that seemed to be the error code I'm getting.